common/scala/src/main/scala/org/apache/openwhisk/core/entity/Parameter.scala

```diff
@@ -252,29 +268,6 @@ protected[core] object Parameters extends ArgNormalizer[Parameters] {
     ParameterValue(Option(v).getOrElse(JsNull), false, None))
   }
 
-  def readMergedList(value: JsValue): Parameters =
```

Comment:
@mcdan by partitioning the arguments into locked and unlocked, I think the approach is simpler. The same information is crossing the message bus, but as two separate maps: the arguments, and the list of ones that are locked.

Reply:
Yea that's a simpler way of looking at it but that changed the wire protocol for kafka right?